test/resources/application.conf

```diff
@@ -14,10 +14,8 @@
 
 play.filters.disabled += "play.filters.csrf.CSRFFilter"
 
-play.http.secret.key = "some_secret"
-
 mongo-async-driver {
```

Comment:
Not sure what this `mongo-async-driver` config is for or if it's used but I noticed that it's been taken out of the main `application.conf` but left in the test `application.conf`?

Is that right? Is it a test only dependency?

Reply:
So I was told to just update the config so that it looked like what current (updated) scaffold is doing when it comes to turning akka into pekko. This is what I could see in hmrc scaffold repository and since this was untouched by us and was just generated by scaffold i amended it to look like what scaffold would currently generate

Reply:
Sounds good to me, cheers :+1: